content/_index.md

```diff
@@ -8,7 +8,11 @@ weight = 1
 
 !['logo-banner'](images/logo_banner_2.png)
 
-Welcome to the **Fractribution** accelerator. This is an accelerator to demonstrate and guide you on how to run fractional attribution analysis on Snowplow data, coined 'Fractribution'. Fractional attribution allows you to attribute the value of a conversion to one or more channels depending on the conversion pathway. As a result, it becomes possible to determine the revenue per channel, as well as ROAS<sup>*</sup> once you have cost data for each marketing channel. Once finished, you will be able to enhance your strategic marketing decisions, such as channel investment and focus, in a more data informed way.
+### Fractribution Accelerator
+
+#### Introduction
+
+This accelerator will guide you to run fractional attribution analysis on Snowplow data. Fractional attribution allows you to attribute the value of a conversion to one or more channels depending on the conversion pathway. As a result, it becomes possible to determine the revenue per channel, as well as ROAS* once you have cost data for each marketing channel. Once finished, you will be able to enhance your strategic marketing decisions, such as channel investment and focus, in a more data informed way.
```

Comment:
We should define out what ROAS is first before using the acronym. So read as:
....as well as Return on Advertising Spend (ROAS*).....